src/components/workflowEditor/EmptyWorkflow.tsx

```diff
@@ -0,0 +1,156 @@
+import { createWorkflow } from './service'
+import { toast } from 'react-toastify'
+import error from '../../assets/icons/misc/errorInfo.svg'
+import { FILTER_NAME_REGEX } from '../ApplicationGroup/Constants'
+import {
+    emptyWorkflowMessage,
+    invalidWorkflowNameError,
+    max30CharError,
+    min3CharError,
+    noWorkflowNameError,
+} from './constants'
+
+export default class EmptyWorkflow extends Component<EmptyWorkflowProps, EmptyWorkflowState> {
+    _inputName: HTMLInputElement
+
+    constructor(props) {
+        super(props)
+        this.state = {
+            name: '',
+            showError: false,
+            loading: false,
+        }
+    }
+
+    componentDidMount() {
+        if (this._inputName) {
+            this._inputName.focus()
+        }
+    }
+
+    handleWorkflowName = (event): void => {
+        this.setState({ name: event.target.value })
+    }
+
+    saveWorkflow = (event): void => {
+        event.preventDefault()
+        this.setState({ showError: true, loading: true })
+        const request = {
+            appId: +this.props.match.params.appId,
+            name: this.state.name,
+        }
+        if (!this.isNameValid().isValid) {
+            this.setState({ loading: false })
+            return
+        }
+        const message = emptyWorkflowMessage
+        const promise = createWorkflow(request)
+        promise
+            .then((response) => {
+                toast.success(message)
+                this.setState({
+                    name: response.result.name,
+                    showError: false,
+                })
+                this.props.onClose()
+                this.props.getWorkflows()
+            })
+            .catch((error: ServerErrors) => {
+                showError(error)
+                this.props.onClose()
+            })
+            .finally(() => {
+                this.setState({ loading: false })
+            })
+    }
+
+    isNameValid(): {errorMsg:string, isValid:boolean} {
+        const name = this.state.name;
+        if (!name) {
+            return {
+                errorMsg: noWorkflowNameError,
+                isValid: false
+            }
+        }
+        if(name.length < 3) {
+            return {
+                errorMsg: min3CharError,
+                isValid: false
+            }
+        }
+        if(name.length > 30) {
+            return {
+                errorMsg: max30CharError,
+                isValid: false
+            }
+        }
+        if (!FILTER_NAME_REGEX.test(name)) {
+            return {
+                errorMsg:invalidWorkflowNameError ,
+                isValid: false
+            }
+        } 
+
+        return {
+            errorMsg: '',
+            isValid: true
+        }
+       
+    }
+
+    render() {
+        const validationStatus = this.isNameValid()
+        return (
+            <DialogForm
+                title={'Create job workflow'}
```

Comment:
It can directly be written as title="Create job workflow"

Reply:
done